src/test/java/org/folio/rest/impl/OaiPmhImplTest.java

```diff
@@ -161,16 +173,91 @@ public void getOaiIdentifiers(TestContext context) {
   }
 
   @Test
-  public void getOaiMetadataFormats(TestContext context) {
-    Async async = context.async();
-
-    RequestSpecification requestSpecification = createBaseRequest();
-    String response = test500WithErrorMessage(requestSpecification, LIST_METADATA_FORMATS_PATH);
+  public void getOaiMetadataFormats(TestContext context) throws JAXBException {
+    logger.info("=== Test Metadata Formats without identifier ===");
+
+    String responseWithoutIdentifier = RestAssured
```

Comment:
there is convenience createBaseRequest method which might be used.

Reply:
Done. Thanks.